internal/validate/kube/utils.go

```diff
@@ -0,0 +1,49 @@
+package kube
+
+import (
+	"fmt"
+	"path/filepath"
+	"reflect"
+	"strings"
+
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+)
+
+func Contains(sl *[]string, t string) bool {
```

Comment:
This probably doesn't need to a be a function as it's only used once. I would move the logic inline to where it is called.